infra/gcp/common.tf

```
@@ -0,0 +1,18 @@
+locals {
+  # Members with view access to private infrastructure.
+  # Anyone here MUST be in the PSWG
+  # Comment indicates github username
+  private_infra_viewers = toset([
+    "howardjohn@google.com",     # howardjohn
+    "johnbhoward96@gmail.com",   # howardjohn
+    "ketihmattix@microsoft.com", # keithmattix
+    "daniel.hawton@solo.io",     #dhawton
```

```suggestion
    "daniel.hawton@solo.io",     # dhawton
```

everyone else has a space, so this was bugs me :D